bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/handler/NetworkHandler.java

```diff
@@ -97,16 +99,16 @@ private void refreshValue(ChannelUID channelUID) {
             case CHANNEL_LATENCY:
             case CHANNEL_DEPRECATED_TIME:
                 presenceDetection.getValue(value -> {
-                    updateState(CHANNEL_LATENCY,
-                            new QuantityType<>(value.getLowestLatency(), MetricPrefix.MILLI(Units.SECOND)));
-                    updateState(CHANNEL_DEPRECATED_TIME, new DecimalType(value.getLowestLatency()));
+                    double latencyMs = durationToMillis(value.getLowestLatency());
+                    updateState(CHANNEL_LATENCY, new QuantityType<>(latencyMs, MetricPrefix.MILLI(Units.SECOND)));
+                    updateState(CHANNEL_DEPRECATED_TIME, new DecimalType(latencyMs));
```

Comment:
Maybe outside scope of this PR, but I'm curious about the history of this "deprecated time" which was introduced in #2580. It doesn't seem to exist?

https://github.com/openhab/openhab-addons/blob/cc0e80280e970ca93973aac2147d9510ed2bace7/bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/NetworkBindingConstants.java#L39-L43

https://github.com/openhab/openhab-addons/blob/cc0e80280e970ca93973aac2147d9510ed2bace7/bundles/org.openhab.binding.network/src/main/resources/OH-INF/thing/thing-types.xml#L13-L17

Since it seems like it was somehow deprecated in 2017, maybe it's "time" to abandon it (pun intended)?

Reply:
It indeed seems to be missing from thing-typis.xml so I think it can be safely removed. :+1: